client/src/components/authentication/AuthenticatedRoute.js

```diff
@@ -3,15 +3,15 @@ import { Redirect, Route } from "react-router";
 
 const AuthenticationCheck = ({ component: Component, user }) => {
   if (user !== null) {
-    return <Component />;
+    return <Component user={user}/>;
   }
   return <Redirect to="/user-sessions/new" />;
 };
 
 const AuthenticatedRoute = ({ component, user, ...rest }) => {
   return (
     <Route
-      // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
We actually want to keep this eslint rule because of the spread operator used on the following line!

Reply:
Oh I thought that was just a comment that needed to be removed.